curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/inpaint.html

```diff
@@ -0,0 +1,129 @@
+<html>
+<head>
+<meta charset="utf-8">
+<title>CVdrawing</title>
+</head>
+<body>
+<p id="status">OpenCV.js is loading...</p>
+<canvas id="canvas" width="640" height="426">
+</canvas>
+<img id="src" src="bird.jpg" style="display:none"/>
+<input type="button" id="myButton" value="Test" disabled=true
+onclick="inpaint()"/>
+
+<script type="text/javascript">
+
+var checker = false;
+var src;
+var mask;
+var dst;
+var p_x = 0;
+var p_y = 0;
+var paint;
+var srcmat;
+var maskmat;
+var dstmat;
+var x;
+var y;
```

Comment:
We have `mask`, `maskmat`. What's the difference?

Reply:
"mask" is unused variable. It is should be deleted.

Reply:
I thinks it's better ti raname "maskmat" to just "mask" then.